WordPress/src/main/AndroidManifest.xml

```diff
@@ -338,10 +338,6 @@
             android:name="org.wordpress.android.ui.people.PeopleManagementActivity"
```

Comment:
Not part of this changeset but, let's remove the `org.wordpress.android` part of the name so to match the rest of the manifest style. Cheers!